packages/api-fetch/src/index.js

```diff
@@ -14,6 +14,26 @@ import namespaceEndpointMiddleware from './middlewares/namespace-endpoint';
 import httpV1Middleware from './middlewares/http-v1';
 import userLocaleMiddleware from './middlewares/user-locale';
 
+/**
+ * Default set of header values which should be sent with every request unless
+ * explicitly provided through apiFetch options.
+ *
+ * @type {Object}
+ */
+const DEFAULT_HEADERS = {
+	Accept: 'application/json, */*;q=0.1',
```

Comment:
Can we add a comment to explain why we add this: (Make sure the backend considers the request, a REST API request.

Reply:
I was (and still am) tempted to drop the fallback value, but had planned to do separately. I'll add a comment which could be generic enough to cover in any case.

Reply:
Added in 64212d2

Relevant Slack conversation: https://wordpress.slack.com/archives/C02QB2JS7/p1541196486514200